apps/meta-data-utils/src/tableQuery.ts

```diff
@@ -284,3 +289,15 @@ export const getTableMetaData = (
   }
   return tableMetaData;
 };
+
+function convertToPascalCase(string: string): string {
```

Comment:
I hope this will be gone soon ;-)

Reply:
yea the idea is to have only one of these ( and then remove it ) , until then its needed to remove the dep of ssr on components